web/packages/hovercards/src/style.scss

```
@@ -151,9 +151,11 @@ $font-sf-pro: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Segoe UI", Robo
 	}
 
 	.gravatar-hovercard__profile-url {
-
-		@include ellipsis(1);
-
+		// webkit-box used by ellipsis mixin
+		// doesn't work well for URLs on Firefox
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
```

`-webkit-box` works fine. The root cause is that `word-break: break-word` behaves slightly differently in Chrome and Firefox. I suggest overriding the property for the class instead:

```suggestion
		@include ellipsis(1);
		/* Ensure consistent word-breaking across browsers. */
		word-break: break-all;
```

---

Ah! Cool! Thanks for the heads up. I'll implement this change 🙂